tools/codegen/dest/lazy_ir.py

```diff
@@ -29,6 +28,8 @@ def node_ctor_inputs(func: LazyIrSchema) -> str:
         else:
             if isinstance(arg.type, BaseCType) and arg.type.type.name == "vector<int64_t>":
                 node_ctor_scalars.append(f"std::vector<int64_t>({arg.name}.begin(), {arg.name}.end())")
+            if isinstance(arg.type, VectorCType):
```

Comment:
hmm what's the difference between the line above and this? Are they both essentially the same thing but represented different ways in python codegen?  If so, should the line above use Helpers:I64List instead of explicit .begin/.end too, for consistency?